distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -220,7 +220,7 @@ public void testCswRegistryStoreUpdate() throws Exception {
                         FederationAdminService.class);
                 federationAdminServiceImpl.updateRegistryEntry(Library.getRegistryNode(regID,
                         "New Node Name",
-                        "2015-02-26T17:16:34.996Z"), destinations);
+                        "2016-02-26T17:16:34.996Z"), destinations);
```

Comment:
The earlier fix caused part of the code to run that wasn't before. This code rejects updates if the modified time of the update is older than the currently existing modified time. There is a 2015 modified date string in the test data. This date just needs to be sometime after that date so i changed it to 2016.